releasewarrior/templates/fennec/release.json.tmpl

```diff
@@ -19,7 +19,7 @@
             {
                 "alias": "pushapk",
                 "description": "run pushapk (temp in-tree relpro)",
-                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/old-how-tos/fennec-temp-relpro.md#kick-off-publish-action-task",
+                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/blob/ff91ed12da49e1bfb1cb0b30020eccf4874607cb/old-how-tos/fennec-temp-relpro.md#publish-release",
```

Comment:
Do we want to pin to certain commit or should we let it point to tip of the master branch?

Reply:
In this case, we should pin it I think. This is an old how-to that doesn't exist on master. That's why I'm using a rev.